docs/references/config/genesis.json.md

```diff
@@ -0,0 +1,211 @@
+number.
+
+| Value type          | string     |
+|:--------------------|:-----------|
+| **Possible values** | &gt; `"0"` |
+
+Setting this parameter limits accepting evidence of malfeasance to blocks with height that are between the last block
+height, and the last block height minus this parameter. Older evidence is discarded.
+
+To clarify: if this parameter is `5` and the last block height is `10`, any evidence with block numbers `6,7,8 or 9` are
+accepted.
+
+## consensus_params.evidence.max_age_duration
+Limit evidence of malfeasance based on how old the evidence is. This parameter limits the evidence based on its
+timestamp.
+
+| Value type          | string     |
+|:--------------------|:-----------|
+| **Possible values** | &gt; `"0"` |
+|                     | `""`       |
+
+Setting this parameter limits accepting evidence of malfeasance to blocks with a timestamp that are between the last
+block's timestamp, and the last block's timestamp minus this parameter. Older evidence is discarded.
+
+To clarify: if this parameter is `86400` (one day in seconds) and the last block's timestamp is
+`2024-03-01T00:00:00.000000Z`, any evidence with block timestamp starting with `2024-02-29T...` are
+accepted.
+
+## consensus_params.evidence.max_bytes
+Limit evidence size.
+
+| Value type          | string                                                                                 |
+|:--------------------|:---------------------------------------------------------------------------------------|
+| **Possible values** | &gt;= `"0"`; &lt;= [consensus_params.block.max_bytes](#consensus_paramsblockmax_bytes) |
+
+## consensus_params.validator.pub_key_types
+List of public key types accepted for consensus validation.
+
+| Value type                           | array of string |
+|:-------------------------------------|:----------------|
+| **Possible values of array strings** | `"ed25519"`     |
+|                                      | `"secp256k1"`   |
+
+Note that the restriction to the specified types is only checked when the network is already running and there are
+changes in the validator set. This means that in the genesis file's [validators](#validators) section, you can define
+both types of consensus keys without restriction.
+
+## consensus_params.version.app
+Consensus parameter set version.
+
+| Value type                           | string |
+|:-------------------------------------|:-------|
+| **Possible values of array strings** | `"0"`  |
+
+This is used by the ABCI application on the chain. It can also be updated on chain by the ABCI application.
+
+## consensus_params.abci.vote_extensions_enable_height
+If you enable vote extensions in the genesis file, you have to decide at which height are the extensions enabled.
+
+| Value type                           | string      |
+|:-------------------------------------|:------------|
+| **Possible values of array strings** | &gt;= `"0"` |
```

Comment:
Does `0` have special meaning?

Reply:
I don't think there's a special value, the parameter is a `uint64` so needs to be greater than '0' and it's the default value

Reply:
actually clarified in the previous comment